test/reporting-framework/base-mysql.report.test.js

```diff
@@ -64,6 +64,7 @@ describe('BaseMysqlReport:', () => {
 
       done();
     });
+    done();
```

Comment:
The done() here means the unit test is longer working, as it will exit, without waiting for the generateReport() call to return.